src/Tribe/Importer/File_Importer_Events.php

```diff
@@ -10,27 +10,55 @@ class Tribe__Events__Importer__File_Importer_Events extends Tribe__Events__Impor
 	protected function match_existing_post( array $record ) {
 		$start_date = $this->get_event_start_date( $record );
 		$end_date   = $this->get_event_end_date( $record );
+		$all_day    = $this->get_boolean_value_by_key( $record, 'event_all_day' );
```

Comment:
No, I don't think so - my note/reply on an earlier commit was wrong (Gustavo was bang on earlier). We're looking for existing records in the db here.